rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy

```diff
@@ -2641,6 +2826,32 @@ class MenuController extends ControllerBase implements ApplicationContextAware{
      * API: /api/jobs, version 1
      */
 
+    @Get(uri='/job/{id}/info')
+    @Operation(
+        method='GET',
+        summary='Get Job Metadata',
+        description='''Get metadata about a specific job.
+
+Authorization required: `read` or `view` for the job.
+
+Since: V18''',
+        tags=['jobs'],
+        parameters = @Parameter(
+            name = "id",
+            description = "Job ID",
+            in = ParameterIn.PATH,
+            required = true,
+            content = @Content(schema = @Schema(implementation = String))
```

Comment:
why this is not defined as `schema = @Schema(type = 'string')`  as the other parameters?